### PR TITLE
feat: rules page added

### PR DESCRIPTION
**Pull request by sharmayatendra**
## Changes

- rules page added for quiz

image

### CODE REVIEW

**Review by tushargoyal812 — COMMENTED** [review]
code looks clean
file structure is good
design looks consistent

File: src/components/Nav/Nav.css [hunk 1]
```diff
@@ -6,4 +6,8 @@
 
 .link {
     color: black;
```
Comment (vikrantkode) [inline src/components/Nav/Nav.css:8]: dont use hardcode colors.

**Review by vikrantkode — APPROVED** [review]
UI is clean 
Code is readable
looking forward for more features

File: src/components/Nav/Nav.css (hunk 1 above)
Comment (sharmayatendra) [reply src/components/Nav/Nav.css:8]: ok

**Pull request merged**